EUROTHRM/EUROTHRM-IOC-01App/Db/devEurothrm.db

```diff
@@ -224,6 +229,83 @@ record(ai, "$(P)OUT_MAX")
   }
 
 ###########
+# Cache & disable ramp when sensor disconneccted
+record(fanout, "$(P)SENSOR_CHECK")
+{
+   field(LNK1, "$(P)SENSOR_CONNECTED")
+   field(LNK2, "$(P)SENSOR_DISCONNECTED")
+}
+
+record(calcout, "$(P)SENSOR_CONNECTED")
+{
+   field(DESC, "Check whether the temperature sensor is reconnected")
+   field(INPA, "$(P)TEMP")
+   field(CALC, "A = 1529")
```

Comment:
Where does this magic number `1529` come from? Can you add a comment. Probably also better to hold it in a PV so we can change it more easily and it isn't duplicated here and in `SENSOR_DISCONNECTED`

Reply:
1529 is the precise temperature value the eurotherm returns when no sensor is detected. I will add a comment for that